src/Hosting/test/FunctionalTests/ShutdownTests.cs

```diff
@@ -28,7 +28,6 @@ public class ShutdownTests : LoggedTest
         [ConditionalFact]
         [OSSkipCondition(OperatingSystems.Windows)]
         [OSSkipCondition(OperatingSystems.MacOSX)]
-        [OSSkipCondition(OperatingSystems.Linux)] // https://github.com/aspnet/AspNetCore-Internal/issues/1687
```

Comment:
Does this test the exit code?

Reply:
Yes, we'd skipped it due to the exit code failure.